qiskit_experiments/library/characterization/analysis/drag_analysis.py

```diff
@@ -41,21 +41,16 @@ class DragCalAnalysis(curve.CurveAnalysis):
         of times that the Drag plus and minus rotations are repeated in curve :math:`i`.
         Note that the aim of the Drag calibration is to find the :math:`\beta` that
         minimizes the phase shifts. This implies that the optimal :math:`\beta` occurs when
-        all three :math:`y` curves are minimum, i.e. they produce the ground state. Therefore,
+        all three :math:`y` curves are minimum, i.e. they produce the ground state. This
+        occurs when
 
         .. math::
 
-            y_i = 0 \quad \Longrightarrow \quad -{\rm amp} \cos(2 \pi\cdot X_i) = {\rm base}
+            {\rm reps}_i * {\rm freq} * (x - \beta) = N
 
-        Here, we abbreviated :math:`{\rm reps}_i\cdot{\rm freq}(x - \beta)` by :math:`X_i`.
-        For a signal between 0 and 1 the :math:`{\rm base}` will typically fit to 0.5. However,
-        the equation has an ambiguity if the amplitude is not properly bounded. Indeed,
-
-        - if :math:`{\rm amp} < 0` then we require :math:`2 \pi\cdot X_i = 0` mod :math:`2\pi`, and
-        - if :math:`{\rm amp} > 0` then we require :math:`2 \pi\cdot X_i = \pi` mod :math:`2\pi`.
-
-        This will result in an ambiguity in :math:`\beta` which we avoid by bounding the amplitude
-        from above by 0.
+        is satisfied with :math:`N` an integer. Note, however, that there is an ambiguity
+        in :math:`\beta` if the amplitude is not properly bounded. We avoid this ambiguity
+        by bounding the amplitude from above by 0.
```

Comment:
Here is a suggestion just to make it more clear what ambiguity is being referred to:

```suggestion
        is satisfied with :math:`N` an integer. Note, however, that this condition
        produces a minimum only when the amplitude is negative. To ensure this is
        the case, we bound the amplitude to be less than 0.
```